sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_set/feature_set_backfill_metadata.py

```diff
@@ -26,15 +26,15 @@ class FeatureSetBackfillMetadata(RestTranslatableMixin):
     def __init__(
         self,
         *,
-        job_id: Optional[str] = None,
+        job_ids: Optional[list] = None,
```

Comment:
For my understanding: why the schema takes a single job_id but here we take a list of job_ids?
Also, type hint for collection should include item type, e.g Optional[List[str]]

Reply:
backend service supports list of ids in the new swagger.

Reply:
Backfill request

take single job id => Which means retry backfilling the interval for what an already executed materialization job ran for.

Backfill response
return multiple job ids (list of all jobs that were submitted as part of this backfill request.

Reply:
I see, so `feature_set_backfill_metadata` is the response object, and `feature_set_backfill_schema`/`feature_set_backfill_request` are user input?

Reply:
you're right.